apis/c/node/src/lib.rs

```diff
@@ -122,6 +122,9 @@ pub enum EventType {
 /// freed yet. The returned `out_ptr` must not be used after
 /// freeing the `event`, since it points directly into the event's
 /// memory.
+/// 
+/// Note: `Out_ptr` is not a null-terminated string. The length of the string 
```

Comment:
Perhaps this can be in a `# Safety` header as convention in Rust code?
```
# Safety
`out_ptr` is not a null-terminated string. The length of the string is given by `out_len`.

Reply:
Thanks for review!